### PR TITLE
Arachni files converted to vulndb format

### PR DESCRIPTION
**Pull request by m0sth8**
PoC

### CODE REVIEW

File: converted/15-directory-listing.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "fix": {
+        "guidance": [
+            "Unless the web server is being utilised to share static and non-", 
+            "sensitive files, enabling directory listing is considered a poor", 
+            "security practice  This can typically be done with a simple", 
+            "configuration change on the server. The steps to disable the directory", 
+            "listing will differ depending on the type of server being used (IIS,", 
+            "Apache, etc.). If directory listing is required, and permitted, then", 
+            "steps should be taken to ensure that the risk of such a configuration", 
+            "is reduced.  These can include:  1. Requiring authentication to access",
```
Comment (andresriancho) [inline converted/15-directory-listing.json]: I believe that new lines were intended here?

File: converted/16-e-mail-address-disclosure.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "id": 16, 
+    "description": [
+        "Email addresses are typically found on \"Contact us\" pages, however,", 
+        "they can also be found within scripts or code comments of the", 
+        "application. They are used to provide a legitimate means of contacting", 
+        "an organisation.  As one of the initial steps in information", 
+        "gathering, cyber-criminals will spider a website and using automated", 
+        "methods collect as many email addresses as possible, that they may", 
+        "then use in a social engineering attack.  Using the same automated",
```
Comment (andresriancho) [inline converted/16-e-mail-address-disclosure.json]: And new line was also here, to separate different paragraphs

File: converted/18-form-based-file-upload.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    "tags": [
+        "file", 
+        "upload"
+    ], 
+    "fix": {
+        "guidance": [
+            "The identified page should at a minimum:  1. Whitelist permitted file", 
+            "types and block all others. This should be conducted     on the MIME", 
+            "type of the file rather than its extension. 2. As the file is", 
+            "uploaded, and prior to being handled (written to the disk) by     the",
```
Comment (andresriancho) [inline converted/18-form-based-file-upload.json]: why so many spaces?

File: converted/18-form-based-file-upload.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        "code (such as PHP) that is then executed when requested by the client."
+    ], 
+    "tags": [
+        "file", 
+        "upload"
+    ], 
+    "fix": {
+        "guidance": [
+            "The identified page should at a minimum:  1. Whitelist permitted file", 
+            "types and block all others. This should be conducted     on the MIME",
```
Comment (andresriancho) [inline converted/18-form-based-file-upload.json]: why so many spaces?

**Comment by andresriancho** [comment]
I believe that the migration tool is doing something nasty with:
- new lines
- multiple spaces

Other than that it all looks good. Question, is there a way to tell the json writer you're using to output the id and title attributes first? It would be great for humans to read, since they get the most important information first.

**Comment by m0sth8** [comment]
Link to convertor PR https://github.com/vulndb/tools/pull/1

**Comment by andresriancho** [comment]
Merging as-is, will do some manual changes to the files, please don't touch them until I finish

**Comment by andresriancho** [comment]
@m0sth8 done with the manual changes to the db files, you can go ahead and improve/change them in any way you want

**Pull request merged**